actiontext/lib/action_text/content.rb

```diff
@@ -105,7 +105,7 @@ def inspect
 
     def ==(other)
       if other.is_a?(self.class)
-        to_s == other.to_s
+        to_html == other.to_html
       end
```

Comment:
This seems like a good change to me.  Although it could result in different behavior if `other` is a subclass of `ActionText::Content` that overrides `to_partial_path`, `to_rendered_html_with_layout`, `to_s`, etc.

If we want to address that, we could compare either `to_html` or `to_s` as appropriate:

```ruby
if self.class == other.class
  to_html == other.to_html
elsif other.is_a?(self.class)
  to_s == other.to_s
end
```

Reply:
@jonathanhefner Thanks for the advice. I pushed a new change that does this!